packages/react-components/react-tree/library/src/components/TreeProvider.tsx

```diff
@@ -21,3 +21,13 @@ export const TreeProvider = (props: React.ProviderProps<TreeContextValue | Subtr
 };
 
 TreeProvider.displayName = 'TreeProvider';
+
+export type TreeRootResetProps = {
+  children?: React.ReactNode;
+};
+
+export const TreeRootReset = (props: TreeRootResetProps) => (
```

Comment:
Let's have a test somewhere to make sure this doesn't regress - it would be a very bad regression for someone using this 😅